bookserver/routers/assessment.py

```diff
@@ -143,3 +166,355 @@ async def get_history(request: Request, request_data: HistoryRequest):
     ]
 
     return make_json_response(detail=res)
+
+
+# Used by :ref:`compareAnswers`
+@router.get("/getaggregateresults")
+async def getaggregateresults(request: Request, div_id: str, course_name: str):
```

Comment:
Nitpick: add return type.

Reply:
We don't have return types on any of the API endpoints as they all end up with some kind of JSON Response that I wouldn't even know how to specify.  I think its fine to leave them all like they are.